spec/database_admin_spec.rb

```diff
@@ -393,6 +401,174 @@
       end
     end
 
+    context "for Swift DB Restore" do
+      let(:example_uri)          { subject.send(:sample_url, 'swift') }
+      let(:uri)                  { URI(File.dirname(example_uri)) }
+      let(:filename)             { File.basename(example_uri) }
+      let(:user)                 { 'foobar' }
+      let(:pass)                 { 'supersecret' }
+      let(:uri_prompt)           { "Enter the location of the remote backup file\nExample: #{example_uri}: " }
+      let(:user_prompt)          { "\n?  Enter the User Name with access to this file.\nExample: 'openstack_user': " }
+      let(:pass_prompt)          { "Enter the password for #{user}" }
+      let(:region_prompt)        { "Enter the OpenStack Swift Region: " }
+      let(:port_prompt)          { "Enter the OpenStack Swift Port: |5000| " }
+      let(:domain_prompt)        { "OpenStack V3 Domain Identifier" }
+      let(:security_protocol_prompt) { "OpenStack Security Protocol\n\n1) SSL without validation\n2) SSL\n3) Non-SSL\n4) None\n\nChoose the openstack security protocol: |3| " }
```

Comment:
_**Pedantic**_ (No need to change):  If you want to make this align without having to move the rest of the `let` statements over, you could change this variable name to `:ssl_prompt` 😉 

(_@NickLaMuro really just felt bad asking to align the the brackets and have the values go further over the 80 char limit without providing a suggestion_)